### ISSUE TITLE
Remove deprecated p3d/AppRunner code

### ISSUE THREAD

**Issue by Moguri** [panda3d/panda3d#516]
With the new deployment code in 1.10, we no longer need the old p3d/AppRunner code. I am aggregating a list of code that can be removed in this issue and will follow up with a Pull Request.

- [ ] direct/src/p3d
- [ ] direct/src/plugin*
- [ ] direct/src/showbase/AppRunnerGlobal.py

**Comment by CFSworks** [comment]
A thousand times yes!

There's some code in `panda/src/iphone` that references runp3d and/or the plugin, and I believe `panda/src/display/subprocess*` also exists to facilitate the plugin. We might want to look into removing those as well.

**Comment by rdb** [comment]
Ideally, we'd still have a way to run old .p3d files for compatibility; there's a lot of old .p3d files floating around the web that would become unplayable.  We could create a little runp3d script that installs Panda3D 1.9 into a virtualenv, mounts the .p3d to the VFS, registers the VFSImporter, and registers a shim AppRunner class.  It doesn't have very high priority for me personally, but maybe someone in the community could work on such a thing?

**Comment by Moguri** [comment]
Maybe it's better to leave this stuff in and then remove it in a major version change?

**Comment by rdb** [comment]
I think it's fine to remove this in master, and consider a possible compatibility p3d runner as a separate issue if someone feels it is needed.

**Comment by Moguri** [comment]
Should the runtime and rtdist build options be removed as part of this?

**Comment by rdb** [comment]
@Moguri Yes, they should.

**Comment by Moguri** [comment]
This should have been closed after #734 was merged; closing now.

**Comment by rdb** [comment]
I noticed that panda/src/dxml (containing TinyXML) isn't used by anything, because it was only used by the plug-in code.

Should we remove it?  There are plenty of alternative XML parsing libraries out there for people that depend on this.
